README.md

```diff
@@ -6,197 +6,22 @@
 Okta ASP.NET middleware
 ========================
 
-This package will add authentication via Okta to your ASP.NET 4.x/Core application. You can follow our instructions below, checkout our examples on GitHub or jump to our quickstart to see how to configure your ASP.NET 4.x/Core web app or your Web API.
-
-* [Quickstart ASP.NET 4.x](https://developer.okta.com/quickstart/#/okta-sign-in-page/dotnet/aspnet4)
-* [Quickstart ASP.NET Core](https://developer.okta.com/quickstart/#/okta-sign-in-page/dotnet/aspnetcore)
-* [ASP.NET 4.x samples](https://github.com/okta/samples-aspnet)
-* [ASP.NET Core samples](https://github.com/okta/samples-aspnetcore)
+This package will add authentication via Okta to your ASP.NET 4.x/Core application. You can follow our instructions below, checkout our examples on GitHub or jump to our quickstarts to see how to configure Okta with your ASP.NET applications. 
```

Comment:
What do you think about `to your ASP.NET or ASP.NET Core application`? Is that better or worse?

Reply:
Now that it is used only here, it makes more sense. Before, I would have to respect this way everywhere.